src/lib/types/minions.ts

```diff
@@ -229,6 +229,7 @@ export interface MahoganyHomesActivityTaskOptions extends MinigameActivityTaskOp
 	xp: number;
 	quantity: number;
 	points: number;
+	tier: string;
```

Comment:
make this `tierID: number`, and give tiers, IDs. Just 1,2,3, etc. It will require a change to fix repeating too probably. otherwise, you will store a string like "Expert (Mahogany Planks)" in the database everytime someone does a trip. a number is smaller to store than even 1 letter, basically.